db/migrate/20230803140953_update_users.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :name, :string
+    add_column :users, :github_name, :string
+    add_column :users, :provider, :string, limit: 50, null: false, default:''
+    add_column :users, :uid, :string, limit: 50, null: false, default:''
+    add_column :users, :avatar_url, :string
```

Comment:
Should the URL not perhaps be *text* instead of *string*?
It is out of our control (this is coming from Github, right?) and might be longer than 256 characters which is the limit for *string* as far as I know.

Reply:
@dewaldreynecke 
Github keeps the avatar url quite clean around 60-70 characters, it may get bigger in the future but I don't see it getting over 256 in the next few years.
This it what we get from github
https://avatars.githubusercontent.com/u/114288585?s=100&v=4

The only time I have seen a url longer than 256 was coping image addresses on google images and the images wouldn't load something about the length of the url tripped a security protocol.
Also I think that because alot of apps grab avatars from Omniauthed accounts they keep them tidy for that purpose.

Reply:
Great! Thanks for checking. Should be fine them.